app/assets/javascripts/shared/components/alert.js

```diff
@@ -0,0 +1,20 @@
+const ALERT_ELEMENT = '#float-alert';
+const TEXT_ALERT_ELEMENT = '#float-alert p';
+const HIDE_TIMEOUT = 5000;
+
+function scheduleHide() {
```

Comment:
You don't have to create a new function for something simple that it's only used once.